constants/megaMenuLinks.ts

```diff
@@ -0,0 +1,337 @@
+type SegmentData = {
+  action: string
+}
+
+/**
+ * Represent a navigation link for the menus of the site
+ */
+interface MegaMenuLink {
```

Comment:
Please, reuse the `NavLink` from `/constants/menuLinks.ts` instead of creating a new interface